lib/order.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'customer' #bringing customer.rb file into order.rb
+require 'csv' #requiring csv ruby module into order.rb
+require 'pry'
+
+class Order
+  @@valid_status = [:pending, :paid, :processing, :shipped, :complete]
+
+  attr_reader :id, :products
+  attr_accessor :products, :customer, :fulfillment_status
+
+  def initialize(id, products, customer, fulfillment_status= :pending)
+    @id = id
+    @products = products
+    @customer = customer
+    raise ArgumentError.new("That is an invalid fulfillment status") unless @@valid_status.include?(fulfillment_status)
+    @fulfillment_status = fulfillment_status
+  end
+
+  #sums up products to calc total and adds 7.5% tax
+  def total
+    @total = 0
+    @products.each_value do |cost|
+      @total += cost
+    end
+    #0.075 represents the sales tax 
+    @total += @total *= 0.075
```

Comment:
This would be more readable as:
`@total += @total * 0.075`

Because the *= does reassignment.